Tools/CISupport/build-webkit-org/steps.py

```diff
@@ -270,7 +270,7 @@ def __init__(self, **kwargs):
     @defer.inlineCallbacks
     def run(self):
         platform = self.getProperty('platform', '*')
-        if platform == 'win':
+        if platform in ('win', 'playstation'):
```

Comment:
Shouldn’t this check  WINDOWS_SHELL_PLATFORM

Reply:
Yeah, that would clearly be better; lemme update.